src/shared/App.less

```diff
@@ -8,6 +8,13 @@
   max-width: @maxConstrainedItemWidth;
   margin: 0 auto;
   display: flex;
+
+  .recently-visited {
+    padding: @default-pad;
+    border: 1px solid @border-color-base;
+    border-radius: @border-radius-base;
+    min-height: 400px;
```

Comment:
It looks weird when empty with this min-height

Reply:
I disagree! the alternative would to have it disappear and that causes jank